PublicMapping/districtbuilder
# Link to help docs

## Overview
Text edits and adds a link to the Getting Started Guide to the landing page empty state. 

### Checklist


### Demo
<img width="1582" alt="Screen Shot 2020-09-11 at 6 48 45 PM" src="https://user-images.githubusercontent.com/5672295/92956837-775ee680-f45f-11ea-99c7-3491beb88033.png">

### Notes
- I think the capitalization looks a little strange in the empty state text—let me know what you think, though.
- We talked about adding this to the Map—this will be done in #394.

## Testing Instructions
- `git pull`
- Log in with a user account that has no maps
- Note that the empty state looks like the above. Click the link in the text under the "Welcome" header. The Getting Started link should open up in a new tab.

Closes #357
